networking/src/Ntp/Client.hs

```diff
@@ -147,6 +147,11 @@ updateStatus cli = updateStatus' cli fn
            , (Wlog.Info, sformat ("Evaluated clock offset "%shown%"mcs") offset)
            )
 
+-- |
+-- Internal commands raised by the send loop.
+data NtpClientCmd = SendRequest
+    deriving Eq
+
```

Comment:
Is that an artifact remaining from some cherry-pick that isn't part of develop anymore?

Reply:
hm - good catch. Let me check that